invenio_app_rdm/theme/webpack.py

```diff
@@ -50,7 +50,7 @@
                 # cookiecutter-invenio-rdm.
                 '../../theme.config$': 'less/theme.config',
                 'themes/rdm': 'less/invenio_app_rdm/theme',
-                '@invenio_theme': 'less/invenio_theme/theme'
+                '@less/invenio_app_rdm': 'less/invenio_app_rdm',
```

Comment:
invenio-theme was defining already an alias to `less/invenio_theme` called `@less/invenio_theme`